Source/Scene/ModelExperimental/PntsLoader.js

```diff
@@ -0,0 +1,523 @@
+      Cartesian3.maximumByComponent(max, position, max);
+    }
+  }
+
+  positions.min = min;
+  positions.max = max;
+}
+
+function makeAttributes(parsedContent, context) {
+  var attributes = [];
+  var attribute;
+  var positions = parsedContent.positions;
+  if (defined(positions)) {
+    computeApproximateExtrema(positions);
+    attribute = makeAttribute(positions, context);
+    attributes.push(attribute);
+  }
+
+  if (defined(parsedContent.normals)) {
+    attribute = makeAttribute(parsedContent.normals, context);
+    attributes.push(attribute);
+  }
+
+  if (defined(parsedContent.colors)) {
+    attribute = makeAttribute(parsedContent.colors, context);
+    attributes.push(attribute);
+  }
+
+  if (defined(parsedContent.batchIds)) {
+    attribute = makeAttribute(parsedContent.batchIds, context);
+    attributes.push(attribute);
+  }
+
+  return attributes;
+}
+
+function makeFeatureMetadata(parsedContent) {
+  var batchLength = parsedContent.batchLength;
+  var pointsLength = parsedContent.pointsLength;
+  var batchTableBinary = parsedContent.batchTableBinary;
+
+  if (defined(batchTableBinary)) {
+    var count = defaultValue(batchLength, pointsLength);
+    return parseBatchTable({
+      count: count,
+      batchTable: parsedContent.batchTableJson,
+      binaryBody: batchTableBinary,
+    });
+  }
+
+  // If batch table is not defined, create a property table without any properties.
+  var emptyPropertyTable = new PropertyTable({
+    name: MetadataClass.BATCH_TABLE_CLASS_NAME,
+    count: pointsLength,
+  });
+  return new FeatureMetadata({
+    schema: {},
+    propertyTables: [emptyPropertyTable],
+  });
+}
+
+function makeComponents(loader, context) {
+  var parsedContent = loader._parsedContent;
+
+  var metallicRoughness = new MetallicRoughness();
+  metallicRoughness.metallicFactor = 0;
+  metallicRoughness.roughnessFactor = 0.9;
+
+  var material = new Material();
+  material.metallicRoughness = metallicRoughness;
+
+  var primitive = new Primitive();
+  primitive.attributes = makeAttributes(parsedContent, context);
+  primitive.primitiveType = PrimitiveType.POINTS;
+  primitive.material = material;
+
+  if (defined(parsedContent.batchIds)) {
+    var featureIdAttribute = new FeatureIdAttribute();
+    featureIdAttribute.propertyTableId = 0;
+    featureIdAttribute.setIndex = 0;
+    primitive.featureIdAttributes = [featureIdAttribute];
+  }
+
+  var node = new Node();
+  node.primitives = [primitive];
+
+  if (defined(parsedContent.rtcCenter)) {
+    node.matrix = Matrix4.fromTranslation(parsedContent.rtcCenter);
+  }
+
+  var scene = new Scene();
+  scene.nodes = [node];
+  scene.upAxis = Axis.Z;
+  scene.forwardAxis = Axis.X;
+
+  var components = new Components();
+  components.scene = scene;
+  components.nodes = [node];
+  components.featureMetadata = makeFeatureMetadata(parsedContent);
+
+  loader._components = components;
+}
+
+PntsLoader.prototype.unload = function () {
+  this._components = undefined;
```

Comment:
Oh yikes, I need to check to see what needs to be done to unload WebGL resources, e.g. the `Buffer`s